src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {!!age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      <p className="Person__partner">
+        {`${
```

Comment:
it isn't best practice have ternary operator inside the other one, so better refactor this code

Reply:
const partner = sex === 'm' ? 'wife': 'husband';

Reply:
`const partner = sex === 'm' ? 'wife': 'husband';`